### PR TITLE
Rv collapse completely

### PR DESCRIPTION
**Pull request by Reettaphant**
## What's changed?

_Detail the main feature of this PR and optionally a link to the relevant issue / card_

Clipboard and front overviews will now collapse completely as per designs. 

## Implementation notes

The clipboard header is now passed to the feed container as a right hand component so that it can be placed to the left of the clipboard. 

The collapse overview and collapse collection controls have now also moved to the fronts component so we can display then nicely on top of the fronts content. 

Made some other changes as well to margins/paddings but I think the layout makes more sense now and the components which should be in in control or margins around them are now doing so. 

Screenshot 2019-05-31 at 08 42 59

Screenshot 2019-05-31 at 08 27 03

_Include any specific areas you want to highlight for review that you feel might be worthy of discussion (i.e. any non-obvious decisions you've made)_

### General
- [ ] 🤖 Relevant tests added
- [ ] ✅ CI checks / tests run locally
- [ ] 🔍 Checked on CODE

### Client
- [ ] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [ ] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [ ] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

**Comment by RichieAHB** [comment]
Just from the screenshots, have we lost some padding around between the clipboard and the first front or is it partly scrolled?

**Comment by Reettaphant** [comment]
@RichieAHB Good catch, I can't remember why I removed it in the first place anymore! The collapse overview is still looks like it's not collapsing completely if the front name is too long because there is too much space in the header. I'm going to make a card to implement the designs to replace the header button text with icons to fix this.

File: client-v2/src/components/Clipboard.tsx [hunk 1]
```diff
@@ -153,112 +133,99 @@ class Clipboard extends React.Component<ClipboardProps> {
...
-              <Root
-                id="clipboard"
-                data-testid="clipboard"
-                style={{ display: 'flex', flex: 1 }}
-              >
-                <ClipboardLevel
-                  onMove={this.handleMove}
-                  onDrop={this.handleInsert}
+      <React.Fragment>
+        {this.props.isClipboardOpen && (
```
Comment (jonathonherbert) [inline client-v2/src/components/Clipboard.tsx]: Super minor and abstract point that probably isn't worth changing in this PR -- is this where we'd want to add this condition? It makes our clipboard component unusable elsewhere. As a rule of thumb I've found that if you're writing a conditional at the root of a component's render fn, it probably belongs somewhere further up the tree. I'll stop writing now ahhh

File: client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx [hunk 2]
```diff
@@ -112,21 +118,24 @@ class SearchInput extends React.Component<SearchInputProps, SearchInputState> {
...
+              searchTermsExist={this.searchTermsExist}
+              onDisplaySearchFilters={this.handleDisplaySearchFilters}
+              onKeyUp={e => {
+                if (e.keyCode === 13) {
+                  this.hideSearchFilters();
+                }
+              }}
+            />
+          </TextInputContainer>
+          {rightHandContainer}
```
Comment (jonathonherbert) [inline client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx]: 👍

**Comment by jonathonherbert** [comment]
Merging this in @Reettaphant's absence, as discussed 👍

**Pull request merged**